conformance/fhir-ig-us-core/CHANGELOG

```diff
@@ -1,5 +1,23 @@
-Changed constraint provenance-1 in StructureDefinition-us-core-provenance.json from: ($this.agent.who.resolve().is Practitioner or Device) implies exists() to: ($this.agent.who.resolve().is(Practitioner) or $this.agent.who.resolve().is(Device)) implies exists() to correct a FHIRPath syntax error
+# Version 1
+- Changed constraint provenance-1 in StructureDefinition-us-core-provenance.json from: ($this.agent.who.resolve().is Practitioner or Device) implies exists() to: ($this.agent.who.resolve().is(Practitioner) or $this.agent.who.resolve().is(Device)) implies exists() to correct a FHIRPath syntax error
     See: https://jira.hl7.org/browse/FHIR-27065 for more information
+- Changed SearchParameter-us-core-procedure-date.json from Procedure.occurence to Procedure.performed
+    See: https://jira.hl7.org/browse/FHIR-27887 for more information
```

Comment:
unless you repeated the same set of changes, i wonder if we could just scrap this and go with whatever changes you made to STU3.1.1 artifacts

Reply:
Agree. Changes to the version that currently curate should be sufficient.

Reply:
10-4.  I'll clean this up.

Reply:
Fixed